src/page-modules/assistant/layout.tsx

```diff
@@ -187,6 +199,20 @@ function AssistantLayout({ children, tripQuery }: AssistantLayoutProps) {
                     data={transportModeFilter}
                     onChange={onTransportFilterChanged}
                   />
+
+                  <Search
+                    label={t(PageText.Assistant.search.input.via)}
+                    placeholder={t(PageText.Assistant.search.input.placeholder)}
+                    onChange={onViaSelected}
+                    selectedItem={tripQuery.via ?? undefined}
+                    autocompleteFocusPoint={tripQuery.via ?? undefined}
+                    button={
+                      <ClearButton
+                        className={style.searchInputButton}
+                        onClear={onClear}
+                      />
```

Comment:
I think this should only be visible when there is a search active

Reply:
Is the idea then that the via input field only becomes visible after the user has clicked on the headline in the filter section?